### PR TITLE
feat: add install script fo avahi daemon

### PR DESCRIPTION
**Pull request by kulti**
Enapter Gateway uses mdns to resolve Gateway by hostname in local network.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,3 +67,41 @@ alerts using `alerts.clear`.
...
+### Note about Enapter Gateway
+
+If you are using [Enapter Gateway](https://handbook.enapter.com/software/gateway_software/), you should use debian base Docker image and run avahi to resolve gateway address:
+```diff
+-FROM python:3.10-alpine3.16
++FROM python:3.10-buster
+
+WORKDIR /app
++
++RUN curl -sSfL https://raw.githubusercontent.com/Enapter/python-sdk/main/extenstions/avahi-debian/install.sh | sh -s
```
Comment (kulti) [inline README.md:97]: I have no good idea to versioning this. This repo tag is related to sdk, not this script. Probably we should store this script separately to manage it versions.

Comment (rnovatorov) [reply README.md:99]: This is fine. I have created an [issue](https://github.com/Enapter/python-sdk/issues/6) to come back to this later.

File: extenstions/avahi-debian/install.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+# allow hostnames with more labels to be resolved. so that we can
+# resolve node1.mycluster.local.
+# (https://github.com/lathiat/nss-mdns#etcmdnsallow)
+echo '*' > /etc/mdns.allow
+
+# Configure NSSwitch to use the mdns4 plugin so mdns.allow is respected
+sed -i "s/hosts:.*/hosts:          files mdns4 dns/g" /etc/nsswitch.conf
+printf "[server]\nenable-dbus=no\n" >> /etc/avahi/avahi-daemon.conf
+chmod 777 /etc/avahi/avahi-daemon.conf
```
Comment (rnovatorov) [inline extenstions/avahi-debian/install.sh:14]: Why are `777`s required here and below? Can we somehow get along using stricter file permissions?

Comment (kulti) [reply extenstions/avahi-debian/install.sh]: `755` should be enough. Fixed.

**Pull request merged**